web-ts/package.json

```diff
@@ -0,0 +1,150 @@
+{
+  "name": "web-ts",
+  "version": "0.1.0",
+  "private": true,
+  "engines": {
+    "node": ">=6.9",
+    "npm": ">=3.8"
+  },
+  "scripts": {
+    "start": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack-dev-server --config tools/webpack.config.ts --progress --env.config_name=integreat-test-cms --env.dev_server",
```

Comment:
`cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\"` most ugly workaround for webpack.config.**ts**
Read more here in the [webpack-docu](https://webpack.js.org/configuration/configuration-languages/#typescript) or [this webpack-cli issue](https://github.com/webpack/webpack-cli/issues/2458)